pkg/cedar/core/exchange.go

```diff
@@ -0,0 +1,83 @@
+package cedarcore
+
+import (
+	"context"
+	"time"
+
+	"github.com/guregu/null/zero"
+
+	"github.com/cmsgov/easi-app/pkg/appcontext"
+	"github.com/cmsgov/easi-app/pkg/cedar/core/gen/client/exchange"
+	"github.com/cmsgov/easi-app/pkg/models"
+)
+
+// GetExchangesBySystem fetches a list of CEDAR exchange records for a given system
+func (c *Client) GetExchangesBySystem(ctx context.Context, cedarSystemID string) ([]*models.CedarExchange, error) {
+	if !c.cedarCoreEnabled(ctx) {
+		appcontext.ZLogger(ctx).Info("CEDAR Core is disabled")
+		return []*models.CedarExchange{}, nil
+	}
+
+	// Construct the parameters
+	params := exchange.NewExchangeFindListParams()
+	params.SetSystemID(cedarSystemID)
```

Comment:
I think this implementation differs from the other CEDAR Core methods in that this one seems to take the CEDAR System ID verbatim, rather than relying on the System Cache to get the ICT Object ID (which is what's used in other endpoints)

https://github.com/CMSgov/easi-app/blob/7ebc3472f9c742d0e585d4442afad19205174e37/pkg/cedar/core/url.go#L23-L30

Reply:
This means that if you get a list of IDs from the `cedarSystems` query, you won't be able to use those here (unless you get the `versionId`, but I don't think the FE is doing that)

Reply:
@ClayBenson94 This is fixed with the change I just pushed